src/pipecat/adapters/schemas/direct_function.py

```diff
@@ -1,3 +1,17 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+"""Direct function wrapper utilities for LLM function calling.
+
+This module provides utilities for wrapping "direct" functions that handle LLM
+function calls. Direct functions have their metadata automatically extracted
+from function signatures and docstrings, allowing them to be used without
```

Comment:
Suggestion: "allowing them to be used without accompanying configurations (as
`FunctionSchema`s or in provider-specific formats)."

This matches the wording elsewhere is and is more grokkable at a glance.